src/test/java/co/rsk/federate/FedNodeRunnerTest.java

```diff
@@ -67,10 +74,16 @@ public void setUp() throws IOException, HSMClientException {
         when(constants.getBridgeConstants()).thenReturn(bridgeConstants);
         when(bridgeConstants.getBtcParamsString()).thenReturn(NetworkParameters.ID_REGTEST);
 
-        HSMBookKeepingClientProvider hsmBookKeepingClientProvider = mock(HSMBookKeepingClientProvider.class);
+        int hsmVersion = 3;
+        HSMClientProtocol protocol = mock(HSMClientProtocol.class);
+        when(protocol.getVersion()).thenReturn(3);
```

Comment:
Why not pass to the return the `hsmVersion` variable instead of the `3` literal?

Reply:
You are right. Thanks, updated